### PR TITLE
add docs for ff.cloud & billing

### PR DESCRIPTION
**Pull request by sammachin**

### CODE REVIEW

File: docs/cloud/README.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+FlowForge Cloud is a hosted service allowing users to sign-up and start creating teams and projects without having to install and manage their own instance of FlowForge.
+The [Concepts](../user/concepts.md) remain the same, however FlowForge Inc. is the administrator of the platform.
+
+## Billing
+Customers are billed at the team level for each project they create. This is a recurring monthly charge.
+See the [Billing](./billing.md) page for more detailed answers about billing.
+
+## Support
+Customers can get support by emailing [support@flowforge.com], we presently only offer support for the flowforge application and your account, any issues relating to Node-RED such as your flows or a 3rd party node should be raised in the appropriate community forum, for example [https://discourse.nodered.org/] or the GitHub project of the third party node.
```
Comment (ZJvandeWeg) [inline docs/cloud/README.md:11]: Missing link location for our email address, did you intent to create a mailto://?

Comment (ZJvandeWeg) [reply docs/cloud/README.md:11]: Same for the discourse link.

File: docs/cloud/README.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+The file system within a project is not persistent, therefore users should avoid writing anything to this. The Node-RED Nodes for file in and out have been removed. Other 3rd party nodes _may_ attempt to access the filesystem with unpredictable results.
+
+## Network Connections
+### HTTP(S) & Websockets
+Projects expose an HTTPS interface on port 443 with each project having its own hostname (example.flowforge.cloud), Plain HTTP requests to port 80 will receive a redirect to HTTPS on port 443.
+You MUST connect using the hostname not the IP address to reach your project.
+Websocket connections over SSL (wss:) are also supported.
+
+### TCP and UDP
+Incoming TCP or UDP connections will not work, the TCP and UDP Nodes have therefore been removed
```
Comment (ZJvandeWeg) [inline docs/cloud/README.md:23]: Why? I'm confused here, and think others will also not understand. HTTP is build on TCP, why does HTTP work, WebSockets, but not TCP?

Same for UDP really.

File: docs/cloud/README.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+### MQTT
+MQTT Connections to an external broker using the standard MQTT nodes will work fine as the connection is initiated by Node-RED.
+
+### IP Addresses
+As mentioned in HTTP above incoming connections MUST be to the hostname not the IP address, In addition the Source IP address of any outgoing connections cannot be guaranteed and may change at any time, therefore this should not be used to validate requests.
+
+
```
Comment (ZJvandeWeg) [inline docs/cloud/README.md:31]: Please trim the end of the file.

File: docs/cloud/README.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+Websocket connections over SSL (wss:) are also supported.
+
+### TCP and UDP
+Incoming TCP or UDP connections will not work, the TCP and UDP Nodes have therefore been removed
+
+### MQTT
+MQTT Connections to an external broker using the standard MQTT nodes will work fine as the connection is initiated by Node-RED.
+
+### IP Addresses
+As mentioned in HTTP above incoming connections MUST be to the hostname not the IP address, In addition the Source IP address of any outgoing connections cannot be guaranteed and may change at any time, therefore this should not be used to validate requests.
```
Comment (ZJvandeWeg) [inline docs/cloud/README.md:29]: The comma after IP address needs to be a dot. The I after is a capital letter already.

File: docs/cloud/billing.md [hunk 5]
```diff
@@ -0,0 +1,28 @@
+
+## Payment Methods 
+We will accept payments via credit or debit card only using Stripe as our payment provider
+All payments are process In US Dollars.
```
Comment (ZJvandeWeg) [inline docs/cloud/billing.md:4]: All payments are processed in US Dollars.

File: docs/cloud/billing.md [hunk 6]
```diff
@@ -0,0 +1,28 @@
+
+## Payment Methods 
+We will accept payments via credit or debit card only using Stripe as our payment provider
```
Comment (ZJvandeWeg) [inline docs/cloud/billing.md:3]: End with a dot please.

File: docs/cloud/billing.md [hunk 7]
```diff
@@ -0,0 +1,28 @@
...
+Teams are billed monthly on the anniversary of the team creation. You will receive one bill for each team. Projects are billed in advance for the coming month. 
+If a project is added during the billing cycle then a new charge is made for a full month and a pro-rated credit is then applied to the invoice on the next cycle for the un-used time.
+
+## Removing Projects
+If you delete a project but still have other projects on their account you will receive a prorated credit for the number of whole days remaining in the billing cycle, which is used against a future invoice.
+If you delete your final project you will receive a prorated refund for the number of days remaining.
+The minimum cost for a project that is created and then deleted will be one day.
+
+## Stopped Projects
+Currently stopped projects are still charged as normal because the container is still running. We hope to offer the ability to suspend a project in a future release.
```
Comment (ZJvandeWeg) [inline docs/cloud/billing.md:20]: Link the issue please.

**Review by ZJvandeWeg — CHANGES_REQUESTED** [review]
Couple of typo's here and there, but generally is a good document. Thanks!

File: docs/cloud/README.md (hunk 2 above)
Comment (sammachin) [reply docs/cloud/README.md:23]: Its inbound connections, so HTTP is mapped through using the reverse proxy on the ingress controller but other TCP and UDP ports are not.
Outbound connections should work fine but currently the TCP & UDP nodes are one node with a setting for inbound or outbound so you can't turn off just inbound. If users need or want outbound then we can look to create custom nodes that just have outbound or give them back the existing nodes on the understanding that inbound won't work.

File: docs/cloud/README.md (hunk 1 above)
Comment (sammachin) [reply docs/cloud/README.md:11]: GitHub takes care of formatting emails/urls without any special markdown so have removed the. []

**Review by ZJvandeWeg — APPROVED** [review]

File: .gitignore [hunk 8]
```diff
@@ -10,4 +10,5 @@ docs/jsdoc
 
 *.env
 *.env.local
+.vscode/settings.json
```
Comment (ZJvandeWeg) [inline .gitignore:13]: @hardillb @sammachin Should this file be ignored? Feels like users should have a global ignore file that keeps editor related files.

**Pull request merged**